cartridges/int_cloudinary/cartridge/client/default/js/cloudinaryWidgets.js

```diff
@@ -28,44 +28,37 @@ window.renderCloudinaryGalleryWidget = function () {
 };
 
 window.renderCloudinarySetGalleryWidgets = function () {
-    var cldObj;
-    var cldSetImages;
-
-    $('.cloudinary-set-data-container').each(function () {
-        cldObj = $(this).data('cloudinary');
-        cldSetImages = $(this).data('cloudinary-set-images');
+    document.querySelectorAll('.cloudinary-set-data-container').forEach(el => {
```

Comment:
Please use `document.getElemenetByClassName`
Please use `for (const el of ...)`